app/views/users/show.html.erb

```diff
@@ -0,0 +1 @@
+<h1><%= @user.name%></h1>
```

Comment:
* As stated in the [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#:~:text=View%20files%20should,a%20given%20user) View files should include only placeholders written in plain HTML, e.g Here is a list of posts for a given user